tests/constructor/test_config_structure.py

```diff
@@ -0,0 +1,56 @@
+import sys
+sys.path.append('../../')
+
+import hydra
+from dataclasses import dataclass, field
+from enum import Enum
+from typing import *
+
+from hydra.core.config_store import ConfigStore
+from omegaconf import OmegaConf
+
+from src.constructor.config_structure import ConfigParams, Phase
+
+
+@hydra.main(config_path="configs/", config_name="config.yaml")
+def test_config_load_when_full_config_was_define(config):
+    config_params = ConfigParams(**config)
+    # test if metric phase is enum
+    if isinstance(type(config_params.metrics[0].phases[0]), Phase):
+        print('Failed. Phase is not Enum.')
```

Comment:
This isn't a fail, this is just a warning. If you want the test to fail here, you should use an assert statement